src/event-cache/EventCache.ts

```diff
@@ -179,6 +204,24 @@ export class EventCache {
      * Removes and returns the next batch of events.
      */
     public getEventBatch(flushCandidates = false): RumEvent[] {
+        if (this.config.debug && this.droppedEvent > 0) {
+            const total = this.events.length + this.droppedEvent;
+            InternalLogger.warn(
+                `Dropped ${
+                    this.droppedEvent
+                } of ${total} recently observed events (${(
+                    (this.droppedEvent / total) *
+                    100
+                ).toFixed(
+                    2
+                )}%) due to insufficient in-memory queue. Increase eventCacheSize to ${total} (currently ${
+                    this.config.eventCacheSize
+                }) to avoid data loss.`
```

Comment:
question: Is there a max eventCacheSize we should recommend in the docs? If we have an estimated size for an event e.g. 1 KB we could probably do the math for customers to let them know how much more they can increase it without impacting their web application's performance due to the browser's in-memory queue being too occupied by the RUM data.